pkgs/applications/office/bookletimposer/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchFromGitLab
+, python3
+, intltool
+, pandoc
+, gobject-introspection
+, wrapGAppsHook
+, gtk3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  name = "bookletimposer";
+  version = "0.3.1";
+  src = fetchFromGitLab {
+    domain = "git.codecoop.org";
+    owner = "kjo";
+    repo = "bookletimposer";
+    rev = version;
+    sha256 = "sha256-AEpvsFBJfyqLucC0l4AN/nA2+aYBR50BEgAcNDJBSqg=";
+  };
+
+  patches = [ ./i18n.patch ./configdir.patch ];
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace 'intltool-extract' '${intltool}/bin/intltool-extract' \
+      --replace 'pandoc' '${pandoc}/bin/pandoc'
+  '';
```

Comment:
Please put the tools into nativeBuildInputs instead which works with cross compiling.

Reply:
Interestingly enough I remembered it didn't work and I had to do it. It seems like adding it to `nativeBuildInputs` is actually enough for python to find it.